File: Python/isPrime.py
num = int(input("Enter a Number :"))
if num <= 1:
    print("Not Prime")
else:
    flag = True
    i = 2
    while i <= num//2:
        if num % i == 0:
            flag = False
            break
        i += 1
if flag:
    print("Prime")
else:
    print("Not Prime")

File: java/IsPrime.java
import java.util.Scanner;

public class IsPrime {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int flag = 1;
        System.out.print("Enter a num : ");
        int num = input.nextInt();
        for(int i = 2; i < num/2; i++) {
            if(num % i == 0) {
                flag = 0;
                break;
            }
        }
        System.out.println((flag==1)?"Prime":"Not Prime");
        input.close();
    }
}
